package/MDAnalysis/lib/distances.py

```diff
@@ -114,29 +114,22 @@ def _run(funcname, args=None, kwargs=None, backend="serial"):
 
 from c_distances_openmp import OPENMP_ENABLED as USED_OPENMP
 
-
 def _box_check(box):
     """Take a box input and deduce what type of system it represents based
     on the shape of the array and whether all angles are 90.
 
-    Parameters
-    ----------
-    box : array
-        Box information of unknown format.
-
-    Returns
-    -------
-    boxtype : str
-        * ``ortho`` orthogonal box
-        * ``tri_vecs`` triclinic box vectors
-        * ``tri_box`` triclinic box lengths and angles
-
-    Raises
-    ------
-    TypeError
-        If box is not float32.
-    ValueError
-        If box type not detected.
+    :Arguments:
```

Comment:
I've got a feeling that he might have forked from master not develop?  These seem like regressions to non numpy docstrings